autospec/files.py

```diff
@@ -178,6 +178,11 @@ def push_file(self, filename):
             self.push_package_file(newfn, "attr")
             self.excludes.append(filename)
 
+        # exclude all lib{,32,64}/haswell pkgconfig files
+        hsw_pkgconfig = re.compile(r"/usr/lib(|32|64)/haswell/pkgconfig/.+")
```

Comment:
Could we instead exclude all pkgconfig files not in /usr/lib64/pkgconfig rather than need to keep poking this every time we add another instruction set.

Reply:
You mean something like replacing
```
r"/usr/lib(|32|64)/haswell/pkgconfig/.+"
```
with
```
r"/usr/lib(|32|64)/[a-zA-Z0-9]+/pkgconfig/.+"
```
?